doc/guide/guide-visualization.rst

```diff
@@ -388,7 +388,7 @@ Note that to obtain :math:`\chi` with this method we have to construct a matrix
 Implementation in QuTiP
 -----------------------
 
-In QuTiP, the procedure described above is implemented in the function :func:`qutip.tomography.qpt`, which returns the :math:`\chi` matrix given a density matrix propagator. To illustrate how to use this function, let's consider the :math:`i`-SWAP gate for two qubits. In QuTiP the function :func:`qutip.qip.operations.iswap` generates the unitary transformation for the state kets:
+In QuTiP, the procedure described above is implemented in the function :func:`qutip.tomography.qpt`, which returns the :math:`\chi` matrix given a density matrix propagator. To illustrate how to use this function, let's consider the :math:`i`-SWAP gate for two qubits. In QuTiP the function :func:`qutip.qip.operations.iswap<qutip.qip.operations.gates.iswap>` generates the unitary transformation for the state kets:
```

Comment:
Is the `<>` bit correct? It looks like maybe the intention was just to add a `.gates`?

Reply:
The method iswap works both ways (with or without .gates). So I didn't want to change the shorter version, but had to add a working link to this function. So I used the sphinx "text-to-link" syntax:
`:func:'Text <link>'`.

Reply:
Cool. Let's leave it like this then.